### PR TITLE
Migrate 'inner-blocks-render-appender' e2e tests to Playwright

### PR DESCRIPTION
**Pull request by Mamaduka**
## What?
Part of #38851.

PR migrates `inner-blocks-render-appender.test.js` e2e tests to Playwright.

## Why?
See #38851.

## Testing Instructions
```
npm run test:e2e:playwright -- test/e2e/specs/editor/plugins/inner-blocks-render-appender.spec.js
```

### CODE REVIEW

File: test/e2e/specs/editor/plugins/inner-blocks-render-appender.spec.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+		await editor.insertBlock( {
+			name: 'test/inner-blocks-render-appender',
+		} );
+
+		const customAppender = page.locator( '.my-custom-awesome-appender' );
+
+		// Verify if the custom block appender text is the expected one.
+		await expect(
+			customAppender.getByText( 'My custom awesome appender' )
+		).toBeVisible();
```
Comment (WunderBart) [inline test/e2e/specs/editor/plugins/inner-blocks-render-appender.spec.js]: Can we use the `toHaveText`/`toContainText` assertion here?

```suggestion
		await expect( customAppender ).toContainText( 'My custom awesome appender' );
```

File: test/e2e/specs/editor/plugins/inner-blocks-render-appender.spec.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+		const dynamimcAppender = page.locator( '.my-dynamic-blocks-appender' );
+		const addBlockBtn = dynamimcAppender.getByRole( 'button', {
+			name: 'Add block',
+		} );
+
+		// Verify if the custom block appender text is the expected one.
+		await expect(
+			dynamimcAppender.getByText( 'Empty Blocks Appender' )
+		).toBeVisible();
```
Comment (WunderBart) [inline test/e2e/specs/editor/plugins/inner-blocks-render-appender.spec.js]: Ditto

**Review by WunderBart — APPROVED** [review]
Only 2 non-blocking suggestions. Otherwise, :shipit:!

**Comment by Mamaduka** [comment]
Thanks for the review, @WunderBart 🙇

**Pull request merged**